pkg/mkimage-iso-efi/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM lfedge/eve-alpine:9fb9b9cbf7d90066a70e4704d04a6fe248ff52bb AS build
+FROM lfedge/eve-alpine:c114cf1d3ea51534f061f9aa949beb6ac5c12fb3 AS build
```

Comment:
We've updated the most recent tag for eve-alpine. Can you check if this is the correct/current one?